Cargo.toml

```diff
@@ -2,32 +2,32 @@
 panic = "unwind"
 
 [patch."https://github.com/sora-xor/sora2-common.git"]
-# beefy-light-client = { path = "../sora2-common-2/pallets/beefy-light-client" }
```

Comment:
I think `sora2-common-2` was committed accidentally. I think it is better to remove comment at all and keep it only local. Anyway, it is not a critical issue.